app/models/affinity_seed.rb

```diff
@@ -43,4 +46,46 @@ def related_one(current_issue)
     return related_person if person == current_person
     return current_issue.person if related_person == current_issue.person
   end
+
+  private
+
+  def add_update_affinity_tag
+    affinity_kind_changed = affinity_kind_id_changed? && affinity_kind_id_was
+    related_person_changed = related_person_id_changed? && related_person_id_was
+
+    if affinity_kind_changed
+      tag_name = AffinityKind.find(affinity_kind_id_was).affinity_to_tag
+      inverse_tag_name = AffinityKind.find(affinity_kind_id_was).inverse_of_tag
+      if related_person_changed
+        Person
+          .find(related_person_id_was)
+          .remove_tag(tag_name)
+      else
+        related_person.remove_tag(tag_name)
+      end
+      person.remove_tag(inverse_tag_name)
+    end
+
+    if related_person_changed
+      if affinity_kind_changed
+        Person
+          .find(related_person_id_was)
+          .remove_tag(AffinityKind.find(affinity_kind_id_was).affinity_to_tag)
+      else
+        Person
+          .find(related_person_id_was)
+          .remove_tag(affinity_kind.affinity_to_tag)
+      end
+    end
+
+    return unless affinity_kind.affinity_to_tag
```

Comment:
esto quizas lo puedas mover mas arriba..pq si no hay tag para asignar, no hay tag para eliminar

Reply:
ademas podria existir inverse y no affinity to tag..y no lo estaria asignando

Reply:
Esa linea la dejé ahí por que pensé que pudiera darse el caso de un affinity kind que es asignado a uno que no tiene affinity_to_tag, por ejemplo, un payee que pasa a ser other necesita que el tag sea removido, pero no requiere que sea agregado el tag other.

Reply:
fiajte el otro comment que te puse